source/protocol/lsp-messages.ads

```diff
@@ -1254,35 +1254,65 @@ package LSP.Messages is
       ignoreIfExists : Optional_Boolean;
    end record;
 
+   procedure Write_CreateFileOptions
```

Comment:
It would be nice to have Read subprograms also.

Reply:
Agree. I'll add those.